test/ProgressOnderwijsUtils.Tests/MetaObjectBulkInsertColumnMappingTest.cs

```diff
@@ -0,0 +1,134 @@
+﻿using System;
+using System.Linq;
+using ExpressionToCodeLib;
+using JetBrains.Annotations;
+using Xunit;
+using static ProgressOnderwijsUtils.SafeSql;
+
+namespace ProgressOnderwijsUtils.Tests
+{
+    public sealed class MetaObjectBulkInsertColumnMappingTest : TransactedLocalConnection
+    {
+        static readonly ParameterizedSql testTableName = ParameterizedSql.CreateDynamic(nameof(MetaObjectBulkInsertColumnMappingTest) + "Table");

```

Comment:
Ik ben tegen table-namen via string-concat; gewoon uitschrijven; is niet erg als die out of sync loopt.